### PR TITLE
New HUD icons for gimmick jobs (and exploration crew)

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
Added new HUD icons to replace some annoying grey HUD icons.

image Barber (Barber's symbol)

image Psychiatrist (an exclamation mark in a speech bubble with medical blue)

image Stage Magician (Magician's hat)
 

image VIP (Since VIP is super important as much as Captain, They have a VIP STAR!!! though it's smaller one.)
Here's some comparison if you are worried of some confusion. 
image

image Debtor (Almost no change, but I wanted to give them a slight difference.)

image Exploration Crew (eh, simple somehow.)
@PowerfulBacon I wonder if you're fine with this. I couldn't think any better symbol for the explo crew, I just made this anyway.

## Why It's Good For The Game
As a security crew, They're a bit annoying to recognise which job they really have. I wanted to remove this annoyance.
People will be more valid to recognise which job they have in a glance than before.

## Changelog
:cl:
imageadd: added job HUD icons for Exploration Crew, Psychiatrist, Barber, Debtor(almost no change), Stage Magician, and VIP.
/:cl:

### CODE REVIEW

**Comment by AnCopper** [comment]
I thought a better symbol for the exploration crew would be a shuttle.

**Comment by EvilDragonfiend** [comment]
> I thought a better symbol for the exploration crew would be a shuttle.

Yes, but when I try to draw a shuttle symbol, It ends up some abomination symbol.

**Comment by Archanial** [comment]
Exploration crew is mainly science so purple would be more fitting.

**Comment by EvilDragonfiend** [comment]
Ah, I thought the purple first but changed my mind later. But for the color, let's confirmed by @PowerfulBacon.

**Comment by PowerfulBacon** [comment]
The current exploration crew icon doesn't even work. Also this will conflict with exploration crew

**Comment by PowerfulBacon** [comment]
Also I really like these icons

**Comment by EvilDragonfiend** [comment]
> The current exploration crew icon doesn't even work. Also this will conflict with exploration crew

then I will remove explo crew from the hud icon list and only explo HUD icon image will be uploaded at this moment. I'll take a look at your commits to remove the conflict after this is merged.

**Comment by EvilDragonfiend** [comment]
This is now fine to merge as long as reviewed.

**Comment by PowerfulBacon** [comment]
Will I mean it will commit anyway since both PRs change the job access list.

**Comment by EvilDragonfiend** [comment]
I think I got how to remove the conflict. I will reopen this PR later after solving the conflict.

**Comment by PowerfulBacon** [comment]
Dont worry about the conflicts, they can be easilly resolved when they arise. (Icon conflicts will just need to use this PR over another)

File: code/modules/jobs/access.dm
Comment (ike709) [inline code/modules/jobs/access.dm]:
```suggestion
	return get_all_jobs() + list("Prisoner", "King", "VIP", "Debtor")
```

**Review by ike709 — APPROVED** [review]

**Pull request merged**
